app/src/lib/stats/stats-database.ts

```diff
@@ -187,6 +187,46 @@ export interface IDailyMeasures {
 
   /** The number of times the user opens the "Rebase current branch" menu item */
   readonly rebaseCurrentBranchMenuCount: number
+
+  /**
+   * The number of times the user has opened their external editor from the
+   * suggested next steps view
+   */
+  readonly suggestedStepOpenInExternalEditor: number
```

Comment:
:art: another consistency thought here is that we've used `Count` as a suffix in places for numeric metric. Should we stick to that here too?

```suggestion
  readonly suggestedStepOpenInExternalEditorCount: number
```

We've been less consistent with this pattern in the past. All of these are could be considered count-like metrics but aren't named as such:

 - `branchComparisons`
 - `coAuthoredCommits`
 - `commits`
 - `defaultBranchComparisons`
 - `divergingBranchBannerDismissal`
 - `divergingBranchBannerDisplayed`
 - `divergingBranchBannerInfluencedMerge`
 - `divergingBranchBannerInitatedMerge`
 - `divergingBranchBannerInitiatedCompare`,
 - `dotcomCommits`
 - `enterpriseCommits`
 - `errorWhenSwitchingBranchesWithUncommmittedChanges`
 - `mergesInitiatedFromComparison`
 - `partialCommits`
 - `prBranchCheckouts`
 - `repoWithIndicatorClicked`
  - `repoWithoutIndicatorClicked`
  - `stashEntriesCreatedOutsideDesktop`
  - `unattributedCommits`

Reply:
Honestly I feel like the `Count` suffix is redundant here. Almost all of our metrics are counts of one form or another. I think the pattern of suffixing with count stems from back when we used Graphite and had naming conventions that altered the behavior of how the metrics were persisted. Given that we don't have any consistency here and that we're unlikely to achieve consistency given the complexity of changing these I'm personally fine with leaving the suffix out but if you're hesitant on merging this PR without it I'll make the change.

Reply:
Nah, this isn't a blocker. Happy to leave it as-is.